src/main/java/org/jboss/ejb/client/EJBClientContext.java

```diff
@@ -475,6 +487,20 @@ public void run() {
         }
     }
 
+    public void unregisterNodeEJBReceivers(String nodeName) {
```

Comment:
It is similar to this though : https://github.com/rstancel/jboss-ejb-client/blob/477e82d5e0c4adcfe42f507465c0531e9623c245/src/main/java/org/jboss/ejb/client/ClusterContext.java#L414